src/app/authentication/login.component.html

```diff
@@ -29,11 +29,11 @@ <h2 class="sub-heading-1">Be a prepared volunteer.</h2>
             <p *ngIf="loginForm.controls['password'].hasError('pattern') && (loginForm.controls['password'].dirty)">Password is invalid</p>
         </div>     
         <div class="form-group">
-            <button type="submit" [disabled]="!loginForm.valid" class="form-control btn btn-primary btn-lg">Login</button>
+            <button type="submit" [disabled]="!loginForm.valid" class="btn btn-block btn-primary">Login</button>
```

Comment:
@yukie1412 This fixes the "Login"  text centering issue on the blue button.
As for the page looking mobile rather than like a web app, I was told to follow the mockups that was present in the repository. I think we will be adding a background image/colour later on, this has to be finalised with Peace Corps.